keps/sig-cli/1440-kubectl-events/README.md

```diff
@@ -178,7 +178,7 @@ Once the experimental kubectl events command is implemented, this can be rolled
 
 ##### GA
 
-- [ ] Address all major issues and bugs raised by community members
+- [x] Address all major issues and bugs raised by community members
```

Comment:
`e2e tests` section above says `missing` but we already have tests, can you provide link to test-grid ?